tls/s2n_client_extensions.c

```diff
@@ -429,3 +444,17 @@ static int s2n_recv_client_sct_list(struct s2n_connection *conn, struct s2n_stuf
     /* Skip reading the extension, per RFC6962 (3.1.1) it SHOULD be empty anyway  */
     return 0;
 }
+
+
+static int s2n_recv_client_max_frag_len(struct s2n_connection *conn, struct s2n_stuffer *extension)
+{
+    uint8_t mfl_code;
+    GUARD(s2n_stuffer_read_uint8(extension, &mfl_code));
+    if (mfl_code >= S2N_TLS_MAX_FRAG_LEN_INVALID || mfl_code_to_length[mfl_code] > S2N_TLS_MAXIMUM_FRAGMENT_LENGTH) {
+        S2N_ERROR(S2N_ERR_INVALID_MAX_FRAG_LEN);
+        return 0;
```

Comment:
Thanks for the catch. 

Should I use print instead to let the user know but not terminate handshake?
No max frag len will be set (continue to use default)

`fprintf(stderr, "invalid Max Fragmentation Length encountered"\n");
`